src/controllers/migrateController.ts

```diff
@@ -0,0 +1,62 @@
+import { JsonController, Post, Body } from "routing-controllers";
```

Comment:
Can we move it to standalone console app?

Reply:
Done